regenerate.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+from typing import List
+
+import argparse
+import os
+import subprocess
+
+
+def run(*args) -> str:
+    return subprocess.run(
+        args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf-8')
+
+
+def GatherPosts(posts_dir: str) -> List[str]:
+    post_paths = run('find', posts_dir, '-name',
+                     '*.md').stdout.strip().split('\n')
+    return post_paths
+
+
+def GenerateHtml(pandoc_flags: str, post_paths: List[str]):
+    for post_path in post_paths:
+        output_path = post_path[:-2] + 'html'
+        run('pandoc', '-s', pandoc_flags
```

Comment:
Definitely premature optimization, but we should eventually await these from a separate greenlet/thread each to make it faster to build a bunch of paths.

Reply:
Added a TODO.